plugins/bufbuild/es/v1.8.0/Dockerfile

```diff
@@ -1,11 +1,13 @@
-# syntax=docker/dockerfile:1.4
+# syntax=docker/dockerfile:1.7
 FROM node:20.11.1-alpine3.19 AS build
 WORKDIR /app
 COPY --link package*.json .
-RUN npm ci
+RUN npm ci \
```

Comment:
tangentially related, but should we be running `npm ci` with `NODE_ENV=production` or an `--omit` to avoid installing devDependencies? It definitely won't get similar gains to this, but might trim `node_modules` even further, unless I'm misunderstanding. (not an expert on node, could be very wrong.)

Reply:
Yep - that would be a good idea. However, we set up package.json with a single dependency which is the NPM package for the plugin - for example: https://github.com/bufbuild/plugins/blob/main/plugins/bufbuild/es/v1.8.0/package.json.

Reply:
Ended up taking this one step further and building the plugin with esbuild.